src/components/JobCard/JobCard.tsx

```diff
@@ -1,23 +1,65 @@
+import React, { useState } from "react";
+import ProjectCardModal from "../Modals/ProjectCardModal/ProjectCardModal";
 
+type Job = {
+  title: string;
+  description: string;
+  imageUrl: string;
+};
+
+type JobCardProps = {
+  job: Job;
+};
 
 const JobCard = ({ job }) => {
+  const [open, setOpen] = useState<boolean>(false);
```

Comment:
instead of open, can you use "isProjectCardModalOpen"

Reply:
Done